Tools/CMake/AMReXThirdPartyLibraries.cmake

```diff
@@ -143,7 +143,7 @@ endif ()
 # HYPRE
 #
 if (AMReX_HYPRE)
-    find_package(HYPRE 2.20.0 REQUIRED)
```

Comment:
This is the minimally supported version (not the exact one).
https://cmake.org/cmake/help/latest/command/find_package.html#typical-usage

In dependencies that support `Config` mode, one can specify if major versions are compatible or not (not often used yet in the wild), which could limit the upper version that is supported.

There is a newer feature where one can specify version ranges, or one can just run `find_package(...)` again (w/o `REQUIRED`) for the major version step, if both are supported.